File: 백준/Bronze/2501. 약수 구하기/약수 구하기.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        int K = sc.nextInt();
        
        int count = 0;
        int Ans = 0;
        
        for (int i = 1; i <= N; i++) {
            if (N % i == 0) {
                count++;
            }
            
            if (count == K) {
                Ans = i;
                break;
            }
        }
        System.out.println(Ans);
    }
}

File: 백준/Bronze/2501. 약수 구하기/약수 구하기.cs
using System;

class Q2501
{
    static void Main(String[] args)
    {
        String[] S = Console.ReadLine().Split();
        int N = int.Parse(S[0]);
        int K = int.Parse(S[1]);
        int count = 0;
        int Ans = 0;
        
        for (int i = 1; i <= N; i++)
        {
            if (N % i == 0)
            {
                count++;
            }
            
            if (count == K)
            {
                Ans = i;
                break;
            }
        }
        
        Console.WriteLine(Ans);
    }
}